t/05-scheduler-full.t

```diff
@@ -57,6 +57,9 @@ use DateTime;
 # This test have to be treated like fullstack.
 plan skip_all => "set SCHEDULER_FULLSTACK=1 (be careful)" unless $ENV{SCHEDULER_FULLSTACK};
 
+diag("OPENQA_BASEDIR: $ENV{OPENQA_BASEDIR}");
+diag("OPENQA_CONFIG:  $ENV{OPENQA_CONFIG}");
```

Comment:
Please don't add more unconditional output in the test. If you want some additional information to show up in verbose-only mode use `note`

Reply:
Just a leftover from debugging. I'm removing it.